packages/grafana-runtime/src/unstable.ts

```diff
@@ -9,5 +9,4 @@
  * and be subject to the standard policies
  */
 
-// Dummy export to make it a valid module. Remove when we have real exports.
-export const unstable = {};
+export { useTranslate as useT, setUseTranslateHook } from './utils/i18n';
```

Comment:
think we should just export this as `useTranslate`

Reply:
ugh sorry, i didn't pay enough attention to the F2 rename. you're 100% right.